{"name": "check users bills", "status": "broken", "statusDetails": {"message": "AttributeError: 'Context' object has no attribute 'bills'\n", "trace": "  File \"c:\\users\\wsu\\pycharmprojects\\msw_api\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\wsu\\pycharmprojects\\msw_api\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\accounting_system.py\", line 71, in step_impl\n    for key, value in context.bills.items():\n  File \"c:\\users\\wsu\\pycharmprojects\\msw_api\\venv\\lib\\site-packages\\behave\\runner.py\", line 321, in __getattr__\n    raise AttributeError(msg)\n"}, "steps": [{"name": "Then check users bills", "status": "broken", "statusDetails": {"message": "AttributeError: 'Context' object has no attribute 'bills'\n", "trace": "  File \"c:\\users\\wsu\\pycharmprojects\\msw_api\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\wsu\\pycharmprojects\\msw_api\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\accounting_system.py\", line 71, in step_impl\n    for key, value in context.bills.items():\n  File \"c:\\users\\wsu\\pycharmprojects\\msw_api\\venv\\lib\\site-packages\\behave\\runner.py\", line 321, in __getattr__\n    raise AttributeError(msg)\n"}, "start": 1611567204907, "stop": 1611567204977}], "start": 1611567204906, "stop": 1611567204978, "uuid": "31bb6148-67b5-4b33-b176-e3b4413e2743", "historyId": "6d6a6330480bef18a36c0f6553303094", "fullName": "features/I-AS/II-fin_user.feature:check users bills", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "fin_user"}, {"name": "feature", "value": "FIN permissions"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}